### PR TITLE
[docs] Add status header for iis receiver

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for iis receiver.

### CODE REVIEW

File: receiver/iisreceiver/README.md [hunk 1]
```diff
@@ -23,4 +27,7 @@ The full list of settings exposed for this receiver are documented [here](./conf
 
 ## Metrics
 
-Details about the metrics produced by this receiver can be found in [documentation.md](./documentation.md) 
\ No newline at end of file
+Details about the metrics produced by this receiver can be found in [documentation.md](./documentation.md) 
+
+[beta]: https://github.com/open-telemetry/opentelemetry-collector#beta
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
```
Comment (codeboten) [inline receiver/iisreceiver/README.md:33]:
```suggestion
```

**Review by codeboten — COMMENTED** [review]
This receiver is currently not in any distribution

**Review by codeboten — COMMENTED** [review]
Pinging @mrod1598 @djaglowski, please confirm stability status

**Comment by djaglowski** [comment]
I am comfortable with this being marked as beta.

**Review by djaglowski — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Pull request merged**